packages/cli/src/commands/test.js

```diff
@@ -100,7 +99,7 @@ export const handler = async ({
 
     // Create a test database
     if (sides.includes('api')) {
-      await execa.command(`yarn rw prisma db push --force`, {
+      await execa(`yarn rw prisma db push --force`, {
```

Comment:
The `.command` breaks tests in the contributor workflow. 🤷🏾‍♂️ Probably due to execa resolving wrongly.